src/main/kotlin/no/skatteetaten/aurora/boober/feature/fluentbit/FluentbitConfigurator.kt

```diff
@@ -72,85 +91,119 @@ class FluentbitConfigurator {
             """.trimMargin()
         }
 
         // Input for the log file produced by fluentbit
         // Filters it to stdout
         private val fluentbitLogInputAndFilter: String = """
-        |[INPUT]
-        |   Name             tail
-        |   Path             /u01/logs/fluentbit
-        |   Path_Key         source
-        |   Tag              fluentbit
-        |   Refresh_Interval 5
-        |   Read_from_Head   true
-        |   Key              event
-        |
-        |[FILTER]
-        |   Name             stdout
-        |   Match            fluentbit
+            |[INPUT]
+            |   Name             tail
+            |   Path             /u01/logs/fluentbit
+            |   Path_Key         source
+            |   Tag              fluentbit
+            |   Refresh_Interval 5
+            |   Read_from_Head   true
+            |   Key              event
+            |
+            |[FILTER]
+            |   Name             stdout
+            |   Match            fluentbit
         """.trimMargin()
 
         // Parser filter to assign it to application tag records
         private val timeParserFilter = """
-    |[FILTER]
-    |   Name parser
-    |   Match *-log4j
-    |   Key_Name event
-    |   Parser timeParser
-    |   Preserve_Key On
-    |   Reserve_Data On
-    """.trimMargin()
+            |[FILTER]
+            |   Name parser
+            |   Match *-log4j
+            |   Key_Name event
+            |   Parser timeParser
+            |   Preserve_Key On
+            |   Reserve_Data On
+        """.trimMargin()
 
         // Multiline filter to assign the multiline_parser to application tag records
         private val multilineLog4jFilter = """
-    |[FILTER]
-    |   name multiline
-    |   match *-log4j
-    |   multiline.key_content event
-    |   multiline.parser multiline-log4j
-    """.trimMargin()
+            |[FILTER]
+            |   name multiline
+            |   match *-log4j
+            |   multiline.key_content event
+            |   multiline.parser multiline-log4j
+        """.trimMargin()
+
+        // Parser filters to assign the evalXmlTimeParser to application tag records
+        private val evalXmlTimeParserFilter = """
+            |[FILTER]
+            |   Name parser
+            |   Match *-evalevent_xml
+            |   Key_Name event
+            |   Parser evalXmlTimeParser
+            |   Preserve_Key On
+            |   Reserve_Data On
+            |
+            |[FILTER]
+            |   Name parser
+            |   Match *-ats:eval:xml
+            |   Key_Name event
+            |   Parser evalXmlTimeParser
+            |   Preserve_Key On
+            |   Reserve_Data On
+        """.trimMargin()
+
+        // Multiline filter to assign the XML multiline parser to application tag records
+        private val multilineEvalXmlFilter = """
+            |[FILTER]
+            |   name multiline
+            |   match *-evalevent_xml
```

Comment:
kunne du slått sammen disse to  filterne? går det an å matche på *xml$ f.eks? 
Evt få taggene til å bli mer like ved input blokken

Reply:
Jeg testet med regex lokalt, men det så ikke ut til å fungere.

Reply:
Vi (Øyvind og jeg) tenker også at det er best å unngå at det kan matche med andre sourcetyper enn akkurat disse to.